utilities/data/getStoryData.ts

```diff
@@ -9,12 +9,12 @@ import { getStoryblokApi, StoryblokClient } from '@storyblok/react/rsc';
  * Make sure to not export the below functions otherwise there will be a typescript error
  * https://github.com/vercel/next.js/discussions/48724
  */
-async function getStoryData({ path }: getStoryDataProps): Promise<ISbResult | { data: 404 }> {
+async function getStoryData({ path, isEditor }: getStoryDataProps): Promise<ISbResult | { data: 404 }> {
```

Comment:
This works. My $0.02 would be to default the `isEditor` param to false so that you only have to pass in TRUE for the editor.

Reply:
Sure - will do!